File: Morning Snack/Python/Perfect.py
number = int(input("Enter an integer: "))

sumOfFactorial = 0

for count in range(1, number):
    if number % count == 0:
        sumOfFactorial += count

if sumOfFactorial == number:
    print("This is a perfect number")
else:
    print("This is not a perfect number")

File: Morning Snack/Javascript/Perfect.js
let number = 7

let sumOfFactorial = 0

for(let count = 1; count < number;count++){
 if(number % count == 0){
  sumOfFactorial += count
}
}

if(sumOfFactorial == number){
console.log("This is a perfect number")
}else{
console.log("This is a not perfect number")
}
